src/index.ts

```diff
@@ -7,6 +7,7 @@ import webpack from "webpack";
 import {
   WebpackPluginServe,
   WebpackPluginServeOptions,
+  // @ts-ignore: webpack-plugin-serve types are broken and need an update
```

Comment:
would it be better to add a `modules.d.ts` somehwere where ts can pick it up and just add:

```
declare module 'webpack-plugin-serve'
```

? This would "stub" everything as "any" and we can remove all `ts-ignores` (they are getting a bit much 😅 ).

if the types are corrected, we can add them, remove the stub and don't have to touch anything else.

Reply:
Sure, I can add a stub as well.

Reply:
I added some kind of a stub for the type to avoid the `ts-ignores`. Once the package type has been fixed, then it can be removed and replaced easily enough.